ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__nutanix-4.16-nightly-x86.yaml

```diff
@@ -0,0 +1,54 @@
+build_root:
+  image_stream_tag:
+    name: ci-tools-build-root
+    namespace: ci
+    tag: latest
+images:
+- dockerfile_path: prow/Dockerfile
+  to: ocp-qe-perfscale-ci-nutanix-416
+promotion:
+  to:
+  - namespace: ci
+    tag: latest
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.16"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- always_run: false
+  as: control-plane-24nodes
+  steps:
+    allow_skip_on_success: true
+    cluster_profile: nutanix-qe
+    test:
+    - chain: openshift-qe-control-plane
+    workflow: cucushift-installer-rehearse-nutanix-ipi
+- always_run: false
+  as: data-path-9nodes
```

Comment:
As far as I know, nutanix cluster is only can be installed on `cluster: build01` build farm. cc @sgaoshang to confirm.

Reply:
This is the key, we can create nutanix cluster after adding cluster: build01, thanks!